config/initializers/react_on_rails.rb

```diff
@@ -1,27 +1,79 @@
+  ################################################################################
+  # default is false
+  config.prerender = true
+
+  # default is true for development, off otherwise
+  config.trace = Rails.env.development?
+
+  ################################################################################
+  # SERVER RENDERING OPTIONS
+  ################################################################################
 
-  # Below options can be overriden by passing to the helper method.
-  config.prerender = true # default is false
-  config.trace = Rails.env.development? # default is true for development, off otherwise
+  # If set to true, this forces Rails to reload the server bundle if it is modified
+  config.development_mode = Rails.env.development?
 
   # For server rendering. This can be set to false so that server side messages are discarded.
-  config.replay_console = true # Default is true. Be cautious about turning this off.
-  config.logging_on_server = true # Default is true. Logs server rendering messags to Rails.logger.info
+  # Default is true. Be cautious about turning this off.
+  config.replay_console = true
+
+  # Default is true. Logs server rendering messages to Rails.logger.info
+  config.logging_on_server = true
+
+  config.raise_on_prerender_error = false # change to true to raise exception on server if the JS code throws
+
+  # Server rendering only (not for render_component helper)
+  # You can configure your pool of JS virtual machines and specify where it should load code:
+  # On MRI, use `therubyracer` for the best performance
+  # (see [discussion](https://github.com/reactjs/react-rails/pull/290))
+  # On MRI, you'll get a deadlock with `pool_size` > 1
+  # If you're using JRuby, you can increase `pool_size` to have real multi-threaded rendering.
+  config.server_renderer_pool_size = 1 # increase if you're on JRuby
+  config.server_renderer_timeout = 20 # seconds
+
+  ################################################################################
+  # MISCELLANEOUS OPTIONS
+  ################################################################################
+  # Default is false, enable if your content security policy doesn't include `style-src: 'unsafe-inline'`
+  config.skip_display_none = false
+
+  # This allows you to add additional values to the Rails Context. Implement one static method
+  # called `custom_context(view_context)` and return a Hash.
+  config.rendering_extension = nil
+
+  # The server render method - either ExecJS or NodeJS
+  config.server_render_method = "ExecJS"
 
-  # Server rendering:
-  config.server_renderer_pool_size  = 1   # increase if you're on JRuby
-  config.server_renderer_timeout    = 20  # seconds
+  # Client js uses assets not digested by rails.
+  # For any asset matching this regex, non-digested symlink will be created
+  # To disable symlinks set this parameter to nil.
+  config.symlink_non_digested_assets_regex = /\.(png|jpg|jpeg|gif|tiff|woff|ttf|eot|svg)/
```

Comment:
This is the default value and it's critical if you're using webpack to create CSS assets.